importers/djornl/parser.py

```diff
@@ -266,13 +274,18 @@ def process_file(self, file, remap_fn, store_fn, err_list, validator=None):
         :param validator: (Validator)   jsonschema validator object
 
         """
-        print("Parsing " + file['data_type'] + " file " + file['file_path'])
+        LOGGER.info("Parsing " + file['data_type'] + " file " + file['file_path'])
         file_parser = self.parser_gen(file)
+
+        def add_error(error):
+            LOGGER.error(error)
+            err_list.append(error)
```

Comment:
The downside of doing it this way is that there will be a lot more output from the parser, with more repetition. If there's a whole ton of errors, you're going to have to scroll or search back in the terminal to try to find where the file summary starts. If you've done several runs of the parser and forgot to clear your terminal, this starts becoming a PITA.